tests/bootstrap.php

```diff
@@ -70,6 +70,14 @@
 	$experiments_manager->set_feature_default_state( $experimental_data['name'], $experiments_manager::STATE_ACTIVE );
 }, 10, 2 );
 
+/**
+ * Override shutdown function of the logger, to prevent logging errors during tests.
+ * core/logger/manager.php:266
+ */
+register_shutdown_function( function () {
+	exit();
```

Comment:
```suggestion
	die;
```

we don't have a single `exit()` in our codebase, so it looks weird 😅

Reply:
Removed